src/modules/dom.js

```diff
@@ -0,0 +1,306 @@
+import APILoader from './api_loader.js';
+
+const loader = new APILoader();
+
+let commentSubmitFlag = false;
+
+class DOMManipulator {
+  displayItems = async () => {
+    loader.url = 'https://api.tvmaze.com/shows';
+    const data = await loader.getData();
+    data.forEach((show) => {
+      this.createCard(show);
+    });
+    const count = await this.showCounter(data);
+    const menuText = document.getElementById('show');
+    menuText.innerHTML = `Shows(${count})`;
+  };
+
+  showCounter = async (data) => data.length;
```

Comment:
According to the requirements, the counter for the cards is supposed to be implemented as a separate module and the absolute length of the API should not be used as the counter, instead you are expected to look for specific elements in the DOM (you can utilize querySelectorAll for this) in order to achieve this. Kindly implement this as described in order to stick with the project requirements.